<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductHandlerForm</class>
 <widget class="QWidget" name="ProductHandlerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>630</width>
       <height>340</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="productInfoEnrollTab">
      <attribute name="title">
       <string>tr(&quot;Product information enroll&quot;)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="infoLabel1">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;Input product information.&quot;)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit1</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="nameLabel1">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>tr(&quot;Product name&quot;)</string>
             </property>
             <property name="buddy">
              <cstring>nameLineEdit1</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameLineEdit1"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="priceLabel1">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>tr(&quot;Product price&quot;)</string>
             </property>
             <property name="buddy">
              <cstring>priceLineEdit1</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="priceLineEdit1"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="sortLabel1">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>tr(&quot;Proudct sort&quot;)</string>
             </property>
             <property name="buddy">
              <cstring>sortLineEdit1</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="sortLineEdit1"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget1">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>tr(&quot;Product ID&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product name&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product price&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product sort&quot;)</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="enrollPushButton">
             <property name="text">
              <string>tr(&quot;enroll&quot;)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="productInfoShowTab">
      <attribute name="title">
       <string>tr(&quot;Product information show&quot;)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="infoLabel2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;Current product information show&quot;)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget2">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>tr(&quot;Product ID&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product name&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product price&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product sort&quot;)</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="productInfoSearchTab">
      <attribute name="title">
       <string>tr(&quot;Product information search&quot;)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="infoLabel3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;Input product ID to search product information.&quot;)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="searchLineEdit"/>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget3">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>tr(&quot;Product ID&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product name&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product price&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product sort&quot;)</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="searchPushButton">
             <property name="text">
              <string>tr(&quot;search&quot;)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="productInfoRemoveTab">
      <attribute name="title">
       <string>tr(&quot;Product information remove&quot;)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="infoLabel4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;Select row to remove&quot;)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget4">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>tr(&quot;Product ID&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product name'&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product price&quot;)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Product sort&quot;)</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="removePushButton">
             <property name="text">
              <string>tr(&quot;remove&quot;)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="productInfoModifyTab">
      <attribute name="title">
       <string>tr(&quot;Product information modify&quot;)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="infoLabel5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;Select row to modify&quot;)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QTableWidget" name="tableWidget5">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>470</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>true</bool>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>tr(&quot;Product ID&quot;)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>tr(&quot;Product name&quot;)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>tr(&quot;Product price&quot;)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>tr(&quot;Product sort&quot;)</string>
             </property>
            </column>
           </widget>
           <widget class="QWidget" name="">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="3" column="0">
                <widget class="QLabel" name="sortLabel2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>tr(&quot;Product Sort&quot;)</string>
                 </property>
                 <property name="buddy">
                  <cstring>sortLineEdit2</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="idLineEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="nameLineEdit2"/>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="sortLineEdit2"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="priceLabel2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>tr(&quot;Product Price&quot;)</string>
                 </property>
                 <property name="buddy">
                  <cstring>priceLineEdit2</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="idLabel">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>tr(&quot;Product ID&quot;)</string>
                 </property>
                 <property name="buddy">
                  <cstring>idLineEdit</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="priceLineEdit2"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="nameLabel2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>tr(&quot;Product Name&quot;)</string>
                 </property>
                 <property name="buddy">
                  <cstring>nameLineEdit2</cstring>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_13">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="modifyPushButton">
             <property name="text">
              <string>tr(&quot;modify&quot;)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
